src/pages/projects/ProjectsPage.tsx

```diff
@@ -1,29 +1,34 @@
 import { useTranslation } from 'react-i18next';
 import { useLocation } from 'react-router';
+import { useQuery } from '@tanstack/react-query';
+import { useDispatch } from 'react-redux';
 import { PageSpinner } from '../../components/PageSpinner';
 import { StyledPageContent } from '../../components/styled/Wrappers';
-import { CristinProject } from '../../types/project.types';
-import { useFetch } from '../../utils/hooks/useFetch';
 import { ProjectLandingPage } from './ProjectLandingPage';
+import { fetchProject } from '../../api/cristinApi';
+import { setNotification } from '../../redux/notificationSlice';
 
 const ProjectsPage = () => {
+  const dispatch = useDispatch();
   const { t } = useTranslation();
   const location = useLocation();
   const projectId = new URLSearchParams(location.search).get('id') ?? '';
-  const [project, isLoadingProject, refetchProject] = useFetch<CristinProject>({
-    url: projectId,
-    errorMessage: t('feedback.error.get_project'),
+
+  const projectQuery = useQuery({
+    queryKey: [projectId],
+    queryFn: () => fetchProject(projectId),
+    onError: () => dispatch(setNotification({ message: t('feedback.error.get_project'), variant: 'error' })),
   });
 
   return (
     <StyledPageContent>
-      {isLoadingProject ? (
+      {projectQuery.isLoading ? (
         <PageSpinner aria-label={t('project.project')} />
       ) : (
-        project && <ProjectLandingPage project={project} refetchProject={refetchProject} />
+        projectQuery.data && <ProjectLandingPage project={projectQuery.data} refetchProject={projectQuery.refetch} />
       )}
     </StyledPageContent>
   );
 };
 
-export default ProjectsPage;
+export default ProjectsPage; /*  */
```

Comment:
Ofte lurt å se gjennom egen PR også, da det ofte lurer seg inn sånne ting 😅 
```suggestion
export default ProjectsPage;
```

Reply:
Oops😬